bazel/repository_locations.bzl

```diff
@@ -1201,8 +1201,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Common Expression Language (CEL) C++ library",
         project_desc = "Common Expression Language (CEL) C++ library",
         project_url = "https://opensource.google/projects/cel",
-        version = "da0aba702f44a41ec6d2eb4bbf6a9f01efc2746d",
-        sha256 = "d62b93fd07c6151749e83855157f3f2778d62c168318f9c40dfcfe1c336c496f",
+        version = "e14d338d0f1bc64e54c4275ac5034fe5dedba969",
+        sha256 = "a0fcf8126d4fb8e0e00be79e626cac741e40b4dcef7d549e99f7a7ef6d7c5443",
```

Comment:
Just out-of-curiosity, how do you get this sha256 number?

Reply:
There's probably a better way but I just put something wrong and then copy the right one from bazel error message :)

Reply:
for ref we have a tool that can update this (the sha is for the downloaded tarball)

```console
bazel run  //bazel:\($target) \($dependency) \($version)
```

$target can be `update` or `api-update`

there is also a bot for this here but you need to be a maintainer to use it https://github.com/envoyproxy/envoy/actions/workflows/envoy-dependency.yml

Reply:
Another way (just for reference):
```sh
curl -Lo ~/Downloads/cel-cpp.tar.gz https://github.com/google/cel-cpp/archive/e14d338d0f1bc64e54c4275ac5034fe5dedba969.tar.gz
sha256sum ~/Downloads/cel-cpp.tar.gz
```

Admittedly this requires trusting the version you downloaded is not yet compromised

Reply:
yeah - this is what the tool does

Reply:
On Linux I download the binary and run `sha256sum` against it